src/lib389/lib389/cli_idm/__init__.py

```diff
@@ -90,6 +94,8 @@ def _generic_get(inst, basedn, log, manager_class, selector, args=None):
     if args and args.json:
         o = mc.get(selector, json=True)
         print(o)
```

Comment:
With changes that were made in the last year, I think we can drop the "print" statements completely, and just use log.info().  Same for the changes below this...

Reply:
Thanks for review, I am fixing it here: https://github.com/389ds/389-ds-base/pull/4594